app/Http/Controllers/Playlist/PlaylistsController.php

```diff
@@ -1385,40 +1384,26 @@ public function getPlaylist($user, $playlist_id)
         return $playlist;
     }
 
-    public function getPlaylistTextFilesets($playlist_id)
+    public function getPlaylistTextFilesets(int $playlist_id) : Array
     {
-        $filesets = Arr::pluck(DB::connection('dbp_users')
-            ->select('select DISTINCT(fileset_id) from playlist_items where playlist_id = ?', [$playlist_id]), 'fileset_id');
-
-        $filesets_hashes = DB::connection('dbp')
-            ->table('bible_filesets')
-            ->select(['hash_id', 'id'])
-            ->whereIn('id', $filesets)->get();
-
-        $hashes_bibles = DB::connection('dbp')
-            ->table('bible_fileset_connections')
-            ->select(['hash_id', 'bible_id'])
-            ->whereIn('hash_id', $filesets_hashes->pluck('hash_id'))->get();
-
-        $text_filesets = DB::connection('dbp')
```

Comment:
@bradflood , This query was not considering that the audio fileset belonging to the playlist item had the same `set_size_code` as the plain text fileset